roles/openshift_health_checker/openshift_checks/package_version.py

```diff
@@ -9,6 +9,19 @@ class PackageVersion(NotContainerizedMixin, OpenShiftCheck):
     name = "package_version"
     tags = ["preflight"]
 
+    openshift_to_ovs_version = {
+        "3.6": "2.6",
+        "3.5": "2.6",
+        "3.4": "2.4",
+    }
+
+    openshift_to_docker_version = {
+        "3.1": "1.8",
+        "3.2": "1.10",
+        "3.3": "1.10",
```

Comment:
@brenton I could not find an excluder script in `origin.spec` in the origin repo, however I did see that releases `1.2 - 1.5` define the [required docker version](https://github.com/openshift/origin/blob/release-1.5/origin.spec#L12) as 1.9.1. I am not sure if it is safe to assume the same docker version requirements as ose, but for now, an `origin` deployment type maps to the same required version of docker as an `enterprise` deployment type